Packs/PAN-OS/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "PAN-OS",
     "description": "Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.",
     "support": "xsoar",
-    "currentVersion": "1.6.3",
+    "currentVersion": "1.6.4",
```

Comment:
can be a major

Reply:
Why?, its only a bug fix and documentation that is not represented in XSOAR itself

Reply:
true.